cni/network/network_windows.go

```diff
@@ -192,3 +192,20 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig) []policy.Policy {
 
 	return policies
 }
+
+func getCustomDNS(nwCfg *cni.NetworkConfig) network.DNSInfo {
```

Comment:
Need a placeholder func in linux specific file that returns empty DNSInfo

Reply:
After getting the build error we reevaluated the design and thought it would be better to call this from with in `getNetworkDNSSettings` and `getEndpointDNSSettings` removing the need for the nested if statements and the call to `getCustomDNS` in `network.go`.  We've started this change and it looks a lot cleaner.  Does that sound ok?